tasks/system_probe.py

```diff
@@ -866,7 +866,8 @@ def kitchen_prepare(ctx, kernel_release=None, ci=False, packages=""):
     # test/kitchen/site-cookbooks/dd-system-probe-check/files/default/tests/pkg/ebpf/testsuite
     # test/kitchen/site-cookbooks/dd-system-probe-check/files/default/tests/pkg/ebpf/bytecode/testsuite
     for i, pkg in enumerate(target_packages):
-        target_path = os.path.join(KITCHEN_ARTIFACT_DIR, re.sub("^.*datadog-agent.", "", pkg))
+        regexstr = f"^{re.escape(os.getcwd())}."
+        target_path = os.path.join(KITCHEN_ARTIFACT_DIR, re.sub(regexstr, "", pkg))
```

Comment:
I think we can avoid the regex completely. 
```suggestion
        target_path = os.path.join(KITCHEN_ARTIFACT_DIR, pkg.lstrip(os.getcwd()))
```